Source/UI/SwiftUI Helper Views/InfiniteListView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  InfiniteListView.swift
+//  Planetary
+//
+//  Created by Matthew Lorentz on 10/7/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import SwiftUI
+
+/// https://swiftuirecipes.com/blog/infinite-scroll-list-in-swiftui
+struct InfiniteListView<DataSource, Content>: View where DataSource: InfiniteList, Content: View {
```

Comment:
InfiniteListView and InfiniteStackView sound like they would both do the same thing: show an infinitely scrolling list of cells. It looks like the main difference is that InfiniteListView is scrollable and refreshable? Do you think we could combine these two components into one? If not maybe we could find names that make it more clear how they are different. Some HeaderDoc comments would help disambiguate as well.

Reply:
I added comments to dismbiguate. The main difference is that in the Profile screen we already have a ScrollView with the header and we just need a non scrollable list of messages. That's why I made these two components but avoided repetition by making the InfiniteListView to use the InfiniteStackView inside.